samples/sample6/template.json

```
@@ -21,11 +21,10 @@
         "High": 0.8
       }
     },
-    {
-      "name": "GaussianBlur",
-      "options": {
-        "kSize": [3, 3],
-        "sigmaX": 0
+     {
+      "name": "MedianBlur",
+      "options": {             //replaced GaussianBlur with MedianBlur
+        "kSize": 5
```

Not a valid json with that comment. Try putting it in a key like this(convention): 
```suggestion
      "//": "replaced GaussianBlur with MedianBlur",
      "name": "MedianBlur",
      "options": {             
        "kSize": 5
```